packages/cohort/src/replicator2/cohort_suffix.rs

```diff
@@ -0,0 +1,368 @@
+use std::collections::VecDeque;
+
+use talos_certifier::model::Decision;
+use talos_suffix::{
+    core::{SuffixConfig, SuffixMeta},
+    errors::SuffixError,
+};
+
+use super::model::ReplicatorCandidate2;
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct SuffixItem {
+    /// Data received from Kafka Consumer
+    pub item: ReplicatorCandidate2,
+    pub item_ver: u64,
+    pub decision_ver: Option<u64>,
+    /// Flag to denote if this item has been decided (is_decided is set to true irrespective of the outcome).
+    pub is_decided: bool,
+    pub is_in_flight: bool,
```

Comment:
As we install serially, only the latest is important to us. Do we need `is_in_flight`? 

The `latest_in_flight` prop on the `CohortReplicator` should be enough for filtering the ones we don't need.

Reply:
This items is in progress in GK branch linked above.